Towny/src/main/java/com/palmergames/bukkit/towny/event/executors/TownyActionEventExecutor.java

```diff
@@ -184,12 +184,9 @@ private static boolean isNotCandle(Block block) {
 	}
 	
 	private static boolean isNotFireSpreadBypassMat(Block block) {
-		switch (block.getType()) {
-			case CAMPFIRE:
-				break;
-			default:
-				block = block.getRelative(BlockFace.DOWN);
-		}
+		if (ItemLists.CAMPFIRES.contains(block.getType()))

```

Comment:
This is meant to do line 190 if it is a Campfire, it looks like you might have left out a `!`.